imap_processing/mag/l0/decom_mag.py

```diff
@@ -129,7 +129,7 @@ def generate_dataset(
         direction.astype(str),
         name="direction_label",
         dims=["direction_label"],
-        attrs=attribute_manager.get_variable_attributes("direction_label"),
+        attrs=attribute_manager.get_variable_attributes("direction_label", check_schema=False),
```

Comment:
cool. Glad that is an option in SAMMI!

Do you think this special case of metadata type for `LABL_PTR_i` attribute will be added to SAMMI in the near future? Because I think that's a standard from SPDF and we use it heavily and new mission may need to in the future.

Reply:
Yes, it's next up on my list to add, just have some other priorities to finish first! Long term I definitely want to fix up LABL_PTR_i.